pyproject.toml

```diff
@@ -9,7 +9,7 @@ description ="An extension library for SCADE Python APIs."
 readme="README.rst"
 
 # only 3.7. and 3.10
-requires-python = ">=3.7,!=3.8.*,!=3.9.*,<3.11"
+requires-python = ">=3.7,!=3.8.*,!=3.9.*"
```

Comment:
This may be a requirement on SCADE side. Can you confirm this, @ansjhenry?

Reply:
oh yes, i commented it to test it locally with my python version, sorry i will revert back.